dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ArtifactDownloader.java

```diff
@@ -46,14 +52,41 @@
 
     private final String appVersion = "1.0.0"; //TODO replace with openliberty version?
 
-    private final List<File> downloadedFiles = new ArrayList<File>();
+    private final List<File> downloadedFiles;
 
     private final static Logger logger = InstallLogUtils.getInstallLogger();
 
-    private final ProgressBar progressBar = ProgressBar.getInstance();
-    private static Map<String, Object> envMap = null;
+    private final ProgressBar progressBar;
 
-    public void synthesizeAndDownloadFeatures(List<String> mavenCoords, String dLocation, MavenRepository repository) throws InstallException {
+    private static Map<String, Object> envMap;
+
+    private final String num_threads = System.getProperty("artifactThreads", DEFAULT_THREAD_NUM).toString();
```

Comment:
Default number of threads is set to 8.
If user wants to increase/decrease number of threads, they can do so by setting the JVM_ARG.
ex) ENV JVM_ARGS="-DartifactThreads=5"

Reply:
Looks good, as discussed we will have to get someones blessing for the JVM argument name. Conventionally it would start with the package name that uses it, ie: `com.ibm.ws.install.artifactThreads`